pkgs/development/python-modules/adafruit-nrfutil/default.nix

```
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyserial
+, click
+, ecdsa
+, behave
+}:
+
+buildPythonPackage rec {
+  pname = "adafruit-nrfutil";
+  version = "0.5.3.post16";
+
```

I suggest to disable the module on older Python releases which are EOL as the chance is very low that somebody will test the module.

```suggestion
  version = "0.5.3.post16";
  format = "setuptools";

  disabled = pythonOlder "3.7";
```